packages/lib/isOutOfBounds.tsx

```diff
@@ -250,6 +250,12 @@ export function isTimeViolatingFutureLimit({
       isAfterRollingEndDay,
       endOfRollingPeriodEndDayInBookerTz: periodLimits.endOfRollingPeriodEndDayInBookerTz.format(),
     });
+    if (isAfterRollingEndDay)
+      log.warn("Booking is out of bounds due to rolling period end day.", {
+        formattedDate: dateInSystemTz.format(),
+        isAfterRollingEndDay,
+        endOfRollingPeriodEndDayInBookerTz: periodLimits.endOfRollingPeriodEndDayInBookerTz.format(),
+      });
```

Comment:
Add detailed logs when booking fails due to attempted booking after the rolling period